integration/test.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -ex
+
+_DIR="${BASH_SOURCE%/*}"
+if [[ ! -d "$_DIR" ]]; then _DIR="$PWD"; fi
+
+while ! kubectl wait --for=condition=available deploy/capi-microk8s-bootstrap-controller-manager -n capi-microk8s-bootstrap-system; do
+    echo "Waiting for bootstrap controller to come up"
+    sleep 3
+done
+while ! kubectl wait --for=condition=available deploy/capi-microk8s-control-plane-controller-manager -n capi-microk8s-control-plane-system; do
+    echo "Waiting for control-plane controller to come up"
+    sleep 3
+done
```

Comment:
This assumes that the microk8s capi providers are already deployed. How can we make it so that this test runs against the code of this repo/branch?